### PR TITLE
Allow Client to Retrieve Multiple Validator Statuses

### PR DESCRIPTION
**Pull request by nisdas**
- [x] Allow the grpc Server to return the validator statuses at regular intervals
- [x] Allow the validator clients to log their Validator statuses.
- [x] Add tests for new Method and fix failing tests

### CODE REVIEW

**Comment by terencechain** [comment]
build and tests are failing

File: beacon-chain/rpc/validator_server.go [hunk 1]
```diff
@@ -294,10 +279,111 @@ func (vs *ValidatorServer) ValidatorStatus(
...
+			Status:    &pb.ValidatorStatusResponse{},
+		}
+		dep, eth1BlockNumBigInt, err := vs.beaconDB.DepositByPubkey(ctx, key)
+		if err != nil {
+			return activeValidatorExists, nil, err
+		}
+		if eth1BlockNumBigInt == nil {
+			statusResponses[i].Status = &pb.ValidatorStatusResponse{
+				Status:                 pb.ValidatorStatus_UNKNOWN_STATUS,
+				ActivationEpoch:        params.BeaconConfig().FarFutureEpoch - params.BeaconConfig().GenesisEpoch,
```
Comment (terencechain) [inline beacon-chain/rpc/validator_server.go]: why subtract `GenesisEpoch` here?

Comment (nisdas) [reply beacon-chain/rpc/validator_server.go]: oh this was for readability, following on from this PR #2475

**Comment by nisdas** [comment]
@terencechain fixed build and tests

File: beacon-chain/rpc/validator_server.go (hunk 1 above)
Comment (terencechain) [reply beacon-chain/rpc/validator_server.go]: aha, got it. thanks!

**Pull request merged**